### ISSUE TITLE
MultiVar not working in query type Pull Requests

### ISSUE THREAD

**Issue by Official-James** [grafana/github-datasource#304]
I am trying to use the `$__multiVar` variable in the Pull Requests query without success. The example given has only been for the Issues query. Has this been tested or implemented on other query types?

Below is the variable configuration

![repo-variable](https://github.com/grafana/github-datasource/assets/67149897/74808db1-385c-46f7-a357-fc1604ee2209)

And this is the query I am running.

![datasource-query](https://github.com/grafana/github-datasource/assets/67149897/1abe73a9-c2fc-4972-a27a-05a486a29656)

The result is no data. Run a single repo query shows the results fine, but I would like to see all the Pull Requests from all repos

**Comment by gabor** [comment]
hi @Official-James , as a quick check, does this work when you hardcode the name of the repository in the `Query` field?
i mean writing into it the fixed `repository:repo_name`?

**Comment by Official-James** [comment]
Hi @gabor. Works fine with query `repository:"repo_name"`

**Comment by ivanahuckova** [comment]
Hi @Official-James, I've tried to reproduce your issue following the same setup you have, but in my case with `grafana` repositories it works as expected.
<img width="1501" alt="image" src="https://github.com/grafana/github-datasource/assets/30407135/60fc77b4-9163-4c3f-8515-e9a4c7393d87">
<img width="1046" alt="image" src="https://github.com/grafana/github-datasource/assets/30407135/ed082737-4e96-4d0b-8f1c-417a2cd21392">

Could you please share
- Just to be sure, are you using following query `$__multiVar(repo,$repo)`? In your examples you are using  `$__multiVar(repository,$repo)` which is not a valid github query based on https://docs.github.com/en/search-github/searching-on-github/searching-issues-and-pull-requests
- Could you please check out, if you can reproduce the issue with searching for `grafana` repositories? Basically the same thing I did in screenshots? 
- Could you provide version of Grafana and Github plugin
- Could you please share what are the repository names? Or if that is confidential, could you share obfuscated names (e.g. `granana/grafana` would be `test/test`. Or `confidential/confidential-1-2-3` would be `test/test-1-1-1`.) In this case, I am trying to see if something in name could be causing the issue - for example special characters. 

Thank you!

**Comment by Official-James** [comment]
Thanks, @ivanahuckova. You are correct; my mistake. I have configured it per the documentation (and your comment) and can confirm that the `multiVar` does not work on the **Vulnerabilities** query only. This is why I misconfigured it; I was troubleshooting. I have attached a video confirming that the query works on Pull Requests and not on Vulnerabilities, and using the existing fields works.

https://github.com/grafana/github-datasource/assets/67149897/4af71b5b-cc1c-423a-a993-b26c0f1fb92b

**Comment by ivanahuckova** [comment]
@Official-James While investigating the issue, I realised that GitHub does not support the `query` parameter for `vulnerabilityAlerts` query that we use. Therefore, in that PR, I am removing the query input, as it is misleading.

Additionally, if we want to implement this feature, GitHub would need to support `query` for `vulnerabilityAlerts`. If this would be beneficial for you, I recommend starting by writing a feature request here: https://github.com/orgs/community/discussions?discussions_q=is%3Aopen+vulnerabilityAlerts+.

I have provided more details in PR https://github.com/grafana/github-datasource/pull/323 that fixes the bug with showing the query input that should not be there.
